src/test_dicom.py

```diff
@@ -0,0 +1,38 @@
+from importers import dicom
+from glob import glob
+from PIL import Image as im
+import numpy as np
+
+def main() -> None:
+    dir: str = "C:/Users/Asa Sprow/Downloads/zone-1-low-energy-dicom/Zone 1 - Low Energy DICOM/PAT_4615_18L/Series0001"
```

Comment:
Since this code is going to be merged into `master`, I think we should remove this absolute file path since it only works on @TheSecondReal0's computer. Not sure how we should replace though. Maybe a command line argument?

Reply:
Agree, I kinda left it there in case other people wanted to test it with a random file (didn't wanna spend the time adding a command line argument). This code will eventually be deleted in favor of a better written version in the Visualizer class/package, this file probably shouldn't even be merged into master and should be stored as a test script or something.

I was having issues importing the other file when it wasn't in the same directory, so not sure how to lay it out.

Reply:
Just saw the updated changes to use command line arguments. I agree that this code needs to eventually be rewritten and/or moved but I think it could be useful in laying the groundwork for our testing scripts.